embedded-graphics/src/image/image16bpp.rs

```diff
@@ -140,14 +139,15 @@ mod tests {
             3,
             3,
         )
-        .translate(Coord::new(100, 200));
+        .translate(Point::new(100, 200));
 
-        assert_eq!(image.top_left(), Coord::new(100, 200));
-        assert_eq!(image.bottom_right(), Coord::new(103, 203));
-        assert_eq!(image.size(), UnsignedCoord::new(3, 3));
+        assert_eq!(image.top_left(), Point::new(100, 200));
+        assert_eq!(image.bottom_right(), Point::new(103, 203));
+        assert_eq!(image.size(), Size::new(3, 3));
     }
 
     #[test]
+    #[ignore]
```

Comment:
🙁 ?

Reply:
As I mentioned in the original comment for this PR, I didn't care to much about the tests in the image module because it will probably be replaced by the code in #152 anyway.